NAMESPACE

```diff
@@ -1,6 +1,7 @@
 # Generated by roxygen2: do not edit by hand
 
 export(condition)
+export(enable_gradethis)
```

Comment:
`enable_gradethis()`?
`gradethis_learnr()`?  Kind of goes with the theme of "grade this XX"

Reply:
(I still really like the idea that this function exists!)